unittests/FEXLinuxTests/tests/CMakeLists.txt

```diff
@@ -8,6 +8,11 @@ unset(CMAKE_CXX_FLAGS)
 
 set(GENERATE_GUEST_INSTALL_TARGETS TRUE)
 
+# Use intel masm syntax. ATT style is maddening
```

Comment:
Please let's keep documentation factual instead of using overly emotional language that doesn't convey any extra information.

Reply:
Changed the comment.